From dde6702f20b7c34b091b25580ab19009c9d91b30 Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@ubuntu.com>
Date: Thu, 26 Nov 2020 21:59:39 +0100
Subject: [PATCH 35/38] hurd: Fix partition paths

We have always had an 's' to separate drive number from partition
number.

Signed-off-by: Brian C. Lane <bcl@redhat.com>
---
 libparted/arch/gnu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libparted/arch/gnu.c b/libparted/arch/gnu.c
index d7204cc..507a5ff 100644
--- a/libparted/arch/gnu.c
+++ b/libparted/arch/gnu.c
@@ -805,7 +805,7 @@ gnu_partition_get_path (const PedPartition* part)
 	result = (char*) ped_malloc (result_len);
 	if (!result)
 		return NULL;
-	snprintf (result, result_len, "%s%d", dev_path, part->num);
+	snprintf (result, result_len, "%ss%d", dev_path, part->num);
 	return result;
 }
 
-- 
2.31.1

